app/views/request_mailer/new_response.text.erb

```diff
@@ -10,8 +10,13 @@
 <%= _('When you get there, please update the status to say if the response ' \
       'contains any useful information.' ) %>
 
+<% if @info_request.embargo %>
+<%= _('All responses will be published when the embargo expires, we depend ' \
```

Comment:
We don't actually use "embargo" in the UI – we use "privacy period". Also suggest minor tweak for readability (`s/, we/. We`)

```diff
+ <%= _('All responses will be published when the privacy period expires. We depend ' \
- <%= _('All responses will be published when the embargo expires, we depend ' \
```